Pod/Classes/KolodaView/KolodaView.swift

```diff
@@ -100,6 +114,10 @@ public class KolodaView: UIView, DraggableCardDelegate {
         }
     }
     
+    private func configure() {
+        NSNotificationCenter.defaultCenter().addObserver(self, selector: "layoutDeck", name: UIDeviceOrientationDidChangeNotification, object: nil)
```

Comment:
I suggest you to replace this line of code to a separate method like 'subscribeForNotifications'